### PR TITLE
docs: add zerolog to improve log.md

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

docs

#### What this PR does / why we need it (en: English/zh: Chinese):

en: 为日志文档新增 zerolog 。
zh: add zerolog to improve log.md.

#### Which issue(s) this PR fixes:

None.

### CODE REVIEW

File: content/zh/docs/hertz/tutorials/framework-exten/log.md [hunk 1]
```diff
@@ -106,3 +106,42 @@ func main() {
...
+	"github.com/cloudwego/hertz/pkg/protocol/consts"
+	hertzZerolog "github.com/hertz-contrib/logger/zerolog"
+)
+
+func main() {
+	h := server.Default()
+
+	logger := hertzZerolog.New(
+		hertzZerolog.WithOutput(os.Stdout),     // allows to specify output
+		hertzZerolog.WithLevel(hlog.LevelWarn), // option with log level
```
Comment (rogerogers) [inline content/zh/docs/hertz/tutorials/framework-exten/log.md:130]: 这里warn，改成info吧。下面输出用的info。

File: content/en/docs/hertz/tutorials/framework-exten/log.md [hunk 2]
```diff
@@ -106,4 +106,43 @@ func main() {
...
+	"github.com/cloudwego/hertz/pkg/protocol/consts"
+	hertzZerolog "github.com/hertz-contrib/logger/zerolog"
+)
+
+func main() {
+	h := server.Default()
+
+	logger := hertzZerolog.New(
+		hertzZerolog.WithOutput(os.Stdout),     // allows to specify output
+		hertzZerolog.WithLevel(hlog.LevelWarn), // option with log level
```
Comment (rogerogers) [inline content/en/docs/hertz/tutorials/framework-exten/log.md:131]: 跟下面一样。

File: content/zh/docs/hertz/tutorials/framework-exten/log.md (hunk 1 above)
Comment (L2ncE) [reply content/zh/docs/hertz/tutorials/framework-exten/log.md]: > 这里warn，改成info吧。下面输出用的info。

done

File: content/en/docs/hertz/tutorials/framework-exten/log.md (hunk 2 above)
Comment (L2ncE) [reply content/en/docs/hertz/tutorials/framework-exten/log.md]: > 跟下面一样。

done

**Comment by rogerogers** [comment]
好的，你自己resolve下吧，我好像没权限resolve。

**Review by li-jin-gou — APPROVED** [review]

**Comment by li-jin-gou** [comment]
The hlog documentation also looks like it needs to be optimised, would you like to be responsible for it? @L2ncE

**Comment by L2ncE** [comment]
> The hlog documentation also looks like it needs to be optimised, would you like to be responsible for it? @L2ncE

I'd like to try it.

**Pull request merged**